ee/pages/payment/[uid].tsx

```diff
@@ -103,8 +99,6 @@ export const getServerSideProps = async (context: GetServerSidePropsContext) =>
       booking,
       payment,
       profile,
-      localeProp: locale,
-      ...(await serverSideTranslations(locale, ["common"])),
     },
```

Comment:
We don't need to pass localeProp and serverSideTranslations anymore. That is handled in app-providers.